jax/_src/lax/lax.py

```diff
@@ -2885,6 +2885,9 @@ def precision_attr(precision: Precision) -> ir.ArrayAttr:
 def _dot_general_lower(ctx, lhs, rhs, *, dimension_numbers,
                        precision, preferred_element_type: np.dtype | None,
                        platform: str = "default"):
+  def _is_fp8_mixed_precision_matmul(_lhs_dtypes, _rhs_dtypes):
+      fp8_dtypes = (dtypes.float8_e4m3fn, dtypes.float8_e5m2)
+      return _lhs_dtypes in fp8_dtypes and _rhs_dtypes in fp8_dtypes
```

Comment:
If we do this, should this compare `bits` from `finfo` against 8?

Reply:
Hello David! For Nvidia GPUs, only mixed precision formats of float8_e4m3fn and float8_e5m2 are supported. AMD and Intel GPUs may support F8E5M2FNUZ and F8E4M3FNUZ formats. However, for now, we are focused solely on the float8_e4m3fn and float8_e5m2 formats.